### PR TITLE
Freeze section in project team members.

### PR DESCRIPTION
**Pull request by Himanshunegi0332**
Targets #2245 

### Description
Added a freeze functionality for each team member for each project.

### Checklist:
- [X] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Entities/Project.php [hunk 1]
```diff
@@ -64,6 +64,11 @@ public function getTeamMembersGroupedByEngagement()
         return $this->getTeamMembers()->select('billing_engagement', DB::raw('count(*) as resource_count'))->groupBy('billing_engagement')->get();
     }
 
+    public function getTeamMembersEfforts()
+    {
+        return $this->getTeamMembers()->select('freeze', DB::raw('count(*) as resource_count'))->groupBy('freeze')->get();
```
Comment (gautam071) [inline Modules/Project/Entities/Project.php:69]: What are trying to do here

File: Modules/Project/Config/config.php [hunk 2]
```diff
@@ -16,6 +16,11 @@
         'customer_support' => 'Customer Support',
         'consultant' => 'Consultant'
     ],
+    'efforts' => [
```
Comment (gautam071) [inline Modules/Project/Config/config.php:19]:
```suggestion
    'project_team_member_efforts' => [
```

**Review by gautam071 — CHANGES_REQUESTED** [review]
left some coments

File: Modules/Project/Entities/Project.php (hunk 1 above)
Comment (Himanshunegi0332) [reply Modules/Project/Entities/Project.php]: sorry, written by mistake.

**Review by gautam071 — COMMENTED** [review]
Approved

**Pull request closed without merging**